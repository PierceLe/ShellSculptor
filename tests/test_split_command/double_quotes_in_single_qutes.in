which 'using shlex is so "interestin"'
