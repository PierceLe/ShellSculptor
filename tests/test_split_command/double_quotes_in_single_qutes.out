>> which 'using shlex is so "interestin"'
using shlex is so "interestin" not found

